src/main/java/logisticspipes/config/Configs.java

```diff
@@ -251,6 +254,22 @@ public static void load() {
                 .get(Configuration.CATEGORY_GENERAL, "easterEggs", Configs.EASTER_EGGS, "Do you fancy easter eggs?")
                 .getBoolean(false);
 
+        Configs.MAX_LOGISTICS_FLUID_TRANSPORT_INNER_CAPACITY = Configs.CONFIGURATION
+            .get(
+                Configuration.CATEGORY_GENERAL,
+                "maxLogisticsFluidInnerCapacity",
+                Configs.MAX_LOGISTICS_FLUID_TRANSPORT_INNER_CAPACITY,
+                "Set the maximum amount for a liquid that can be held in a fluid pipe. Default value: 10000 (1000 equals 1 Bucket for normal fluids). If you are playing GTNH this value represents the liters this number (10000 = 10000L).")
+            .getInt();
+
+        Configs.MAX_LOGISTICS_FLUID_TRANSPORT_SIDE_CAPACITY = Configs.CONFIGURATION
+            .get(
+                Configuration.CATEGORY_GENERAL,
+                "maxLogisticsFluidSideCapacity",
+                Configs.MAX_LOGISTICS_FLUID_TRANSPORT_SIDE_CAPACITY,
+                "Set the maximum amount for a liquid that can be held in each attached side a fluid pipe. This value should be half of 'maxLogisticsFluidInnerCapacity' Default value: 5000 (1000 equals 1 Bucket for normal fluids). If you are playing GTNH this value represents the liters this number (10000 = 10000L).")
```

Comment:
```suggestion
                "Sets the maximum amount of a liquid that can be held in each attached side of a fluid pipe. This value should be half of 'maxLogisticsFluidInnerCapacity' Default value: 5000 (1000 equals 1 Bucket for normal fluids). If you are playing GTNH this value represents liters (10000 = 10000L).")
```

> This value should be half of 'maxLogisticsFluidInnerCapacity'

Then does it make sense for this to be configurable at all instead of just `Configs.MAX_LOGISTICS_FLUID_TRANSPORT_INNER_CAPACITY / 2`? If you can see a scenario where this value might be different, then it would probably better to rephrase this to something like:

> Normally, this value is half of 'maxLogisticsFluidInnerCapacity'

Reply:
At last it's how I understood the function and sure, it could simple be a division by 2. Will change this asap after re-adding my GPG Signing Key ;)